src/SFA.DAS.Charities.Import.Functions/Infrastructure/ITimeProvider.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace SFA.DAS.Charities.Import.Functions.Infrastructure
+{
+    public interface ITimeProvider
+    {
+        DateTime Today { get;  }
+    }
+
+    internal class TimeProvider : ITimeProvider
+    {
+        public DateTime Today { get; } = DateTime.Today;
```

Comment:
This is implemented incorrectly. I'd expect to see a constructor that takes in a DateTime object so that Dependency Injection is used correctly

Reply:
I don't want to over complicate this as we are never going to inject anything else.